Mathlib/RingTheory/Localization/Away/Basic.lean

```diff
@@ -79,12 +79,12 @@ noncomputable def lift (hg : IsUnit (g x)) : S →+* P :=
 #align is_localization.away.lift IsLocalization.Away.lift
 
 @[simp]
-theorem AwayMap.lift_eq (hg : IsUnit (g x)) (a : R) : lift x hg ((algebraMap R S) a) = g a :=
+nonrec theorem AwayMap.lift_eq (hg : IsUnit (g x)) (a : R) : lift x hg ((algebraMap R S) a) = g a :=
   lift_eq _ _
```

Comment:
This is more readable if you don't use `nonrec`, but just use the explicit namespace in the proof. It's okay to use nonrec when you want to do fancier things (i.e. use dot notation with the "other" declaration in the proof.